yarn.lock

```diff
@@ -6761,6 +6770,14 @@ react-is@^18.0.0:
   resolved "https://registry.yarnpkg.com/react-is/-/react-is-18.2.0.tgz#199431eeaaa2e09f86427efbb4f1473edb47609b"
   integrity sha512-xWGDIW6x921xtzPkhiULtthJHoJvBbF3q26fzloPCK0hsvxtPVelvftw3zjbHWSkR2km9Z+4uxbDDK/6Zw9B8w==
 
+react@^17.0.2:
+  version "17.0.2"
```

Comment:
So we now have both react 17 and 18 in this repo?

Reply:
We had them both for short time between this PR merge and update to Docusaurus 3, where we collapsed both to one version (18)